app/views/landing_page/blocks/_heading.html.erb

```diff
@@ -3,12 +3,16 @@
   text = block.data["content"]
   path = block.data["path"] || nil
   inverse = options[:inverse] || false
-  margin_bottom = options[:margin_bottom] || 6
+  border_top = block.data["border_top"] || nil
+  padding = block.data["padding"] || false
```

Comment:
These are just exposing options from the gem `heading` component. They're needed on the Press Notices heading and the Share icons heading, and isn't related to cards.

As we need it in multiple places, it made sense to me to allow these options on the heading rather than messing about with other components or new wrapping elements. 

![image](https://github.com/user-attachments/assets/66f6a27e-57b5-4203-b2eb-61b482242881)

Reply:
Thanks for the context. Yes, in that case, I think a new PR to extend the document list block and the sharing block would be good - better to keep all this logic and styling internal to the blocks.